nexus/db-queries/src/db/lookup.rs

```diff
@@ -742,6 +790,18 @@ lookup_resource! {
     primary_key_columns = [ { column_name = "id", rust_type = Uuid } ]
 }
 
+// Helpers for unifying the interfaces around images
+
+pub enum ImageLookup<'a> {
+    ProjectImage(ProjectImage<'a>),
+    SiloImage(SiloImage<'a>),
+}
+
+pub enum ImageParentLookup<'a> {
+    Project(Project<'a>),
+    Silo(Silo<'a>),
+}
+
```

Comment:
This might not really be the best place for these little helper enums, but I wasn't sure where else to put them.